Resources/Prototypes/Entities/Objects/Weapons/Guns/turrets.yml

```
@@ -128,6 +128,48 @@
+            !type:DamageTrigger
+            damage: 200
+          behaviors:
+            - !type:DoActsBehavior
+              acts: [ "Destruction" ]
+        - trigger:
+            !type:DamageTrigger
+            damage: 100
+          # TODO: Construction graph
+          behaviors:
+            - !type:DoActsBehavior
+              acts: [ "Destruction" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)